src/CTA.WebForms2Blazor/Helpers/FilePathHelper.cs

```diff
@@ -31,6 +36,16 @@ public static string AlterFileName(string oldFilePath, string newFileName = null
             return newFileNameWithExtension;
         }
 
+        public static string FilePathToNamespace(string filePath, string projectName)
+        {
+            string res = filePath.RemoveOuterQuotes();
+            res = Path.GetDirectoryName(res);
+
+            return res.IsNullOrEmpty()
+                ? string.Format(FileNameDoesNotContainDirectoryError, filePath)
+                : res.Replace("~", projectName).Replace("/", ".");
+        }
```

Comment:
Looks like this assumes relative pathing is used. Do we have to worry about absolute paths such as `C:\Some\Directory\Etc`?

If so, we should handle that case. If not, we should consider renaming this to "RelativeFilePathToNamespace()" to set expectations

Reply:
So the project name should be the relative path always since thats what is passed in from the very beginning, so I'll just rename the method so its clear